scss/_patterns_card.scss

```diff
@@ -1,62 +1,107 @@
 @import 'settings';
 
+@mixin vf-p-card {
+  %p-card {
+    @extend %bg--x-light;
+    margin-bottom: $spv-inter--scaleable;
+    overflow: auto; // prevent overflow of child margins
+    padding: $spv-intra--scaleable;
+  }
+
+  @include vf-p-card-default;
+  @include vf-p-card-highlighted;
+  @include vf-p-card-overlay;
+  @include vf-p-card-muted;
+  @include vf-p-card-specific-content;
+}
+
 @mixin vf-p-card-default {
   .p-card {
+    @extend %bg--x-light;
+    @extend %is-bordered;
+    @extend %has-round-corners;
     @extend %p-card;
-    border: 1px solid $color-mid-light;
-    padding: $spv-intra--scaleable - $px $spv-intra--scaleable $spv-intra--scaleable;
+    padding: $spv-intra--scaleable - $px;
   }
 }
 
 @mixin vf-p-card-highlighted {
+  %p-card--highlighted {
+    @extend %bg--x-light;
+    @extend %has-highlight;
+    @extend %has-round-corners;
+    @extend %p-card;
+  }
+
   .p-card--highlighted {
     @extend %p-card--highlighted;
   }
 }
 
 @mixin vf-p-card-overlay {
   .p-card--overlay {
-    @extend %p-card;
     background: transparentize($color-x-light, .1);
-    border-radius: 0;
+    color: $color-dark;
+    margin-bottom: $spv-inter--scaleable;
+    overflow: auto;
+    padding: $spv-intra--scaleable;
   }
 }
 
-@mixin vf-p-card {
-  %p-card {
-    background: $color-x-light;
-    border-radius: $border-radius;
-    color: $color-dark;
+@mixin vf-p-card-muted {
+  .p-card--muted {
+    @extend %bg--light;
+    @extend %has-highlight;
+    @extend %has-round-corners;
     margin-bottom: $spv-inter--scaleable;
-    overflow: auto; // prevent overflow of children margins
+    overflow: auto;
     padding: $spv-intra--scaleable;
```

Comment:
Why not extend p-card as it has the styling here.

Reply:
To keep overrides to a minimum. If I extend p-card, that would set background color to white, and then set it to light-grey here.